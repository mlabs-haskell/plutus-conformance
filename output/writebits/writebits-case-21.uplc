[
  [ [ (builtin writeBits) (con bytestring #f4ff) ] (con (list integer) [10]) ]
  (con (list bool) [False])
]