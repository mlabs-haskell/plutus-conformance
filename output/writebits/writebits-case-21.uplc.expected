(con bytestring #f0ff)